src/vid2805-dual-stepper-element.ts

```diff
@@ -0,0 +1,191 @@
+import { html, LitElement, svg } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import { ElementPin } from '.';
+import { mmToPix } from './utils/units';
+
+export type HandShape = 'arrow' | 'plain'; // | 'fancy';
+
+export class StepperHand {
+  /**
+   * length used to control hand length
+   */
+  length = 25;
+
+  /**
+   * colour used to paint the hand
+   */
+  colour = '';
```

Comment:
Let's use `color` to be consistent with the other elements (e.g. the LED, pushbutton, etc.)

Reply:
done